apps/wallet/configs/webpack/webpack.config.common.ts

```diff
@@ -13,7 +14,25 @@ import packageJson from '../../package.json';
 
 import type { Configuration } from 'webpack';
 
-const APP_NAME = 'Sui Wallet';
+function generateDateVersion(patch: number) {
+    const sha = gitRevSync.short();
+    const date = new Date();
+    const version = [
+        String(date.getUTCFullYear()).slice(2),
+        String(date.getUTCMonth() + 1),
+        String(date.getUTCDate()),
+        patch ?? 0,
```

Comment:
`??` shouldn't be necessary right? Patch is a number

Reply:
Good catch, was from an older version of the code, will remove.